### DISCUSSION TITLE
How to take a screenshot!

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [fzyzcjy/flutter_convenient_test#243] [Q&A] [answered]
Is there any way or any method that this package exposes that would allow us to override the screenshot taken?
Sometimes the screenshot lags, especially when dealing with scrolling objects.

**Comment by fzyzcjy** [comment, marked as answer]
Currently it is: https://github.com/fzyzcjy/flutter_convenient_test/blob/92bd62ed218ee822f3633fa2f822b122d6b8b93f/packages/convenient_test/example/integration_test/main_test.dart#L91

When calling that method, it takes a screenshot.

Btw we also have a full video record as well.

**Reply by Rohithgilla12** [reply]
Thanks, that's awesome!

**Reply by fzyzcjy** [reply]
You are welcome!
